packages/templates/nextjs-marketing-demo/src/components/CustomRatingComponentRegistration.tsx

```diff
@@ -0,0 +1,52 @@
+import { Rate } from 'antd';
+type CustomRatingComponentProps = React.ComponentProps<typeof Rate>;
+
+export const CustomRatingComponentRegistration = {
+  component: (props: CustomRatingComponentProps) => {
```

Comment:
since we don't need to modify the props anyway coming in, I think we can just use the Rate component here instead of wrapping it